src/main/kotlin/no/nav/syfo/Bootstrap.kt

```diff
@@ -488,18 +488,27 @@ fun sendInfotrygdOppdatering(
     val personNrPasient = receivedSykmelding.personNrPasient
     val signaturDato = receivedSykmelding.sykmelding.signaturDato.toLocalDate()
     val tssid = receivedSykmelding.tssid
-    val sha256String = sha256hashstring(createInfotrygdBlokk(
+    val sha256StringMedArbeidsgiver = sha256hashstring(createInfotrygdBlokk(
             itfh, perioder.first(), personNrPasient, signaturDato,
-            behandlerKode, tssid, loggingMeta, navKontorNr)
+            behandlerKode, tssid, loggingMeta, navKontorNr, findarbeidsKategori(itfh.healthInformation.arbeidsgiver?.navnArbeidsgiver))
+    )
+    val sha256StringUtenArbeidsgiver = sha256hashstring(createInfotrygdBlokk(
+            itfh, perioder.first(), personNrPasient, signaturDato,
+            behandlerKode, tssid, loggingMeta, navKontorNr, "")
     )
 
     try {
-        val redisSha256String = jedis.get(sha256String)
-        if (redisSha256String != null) {
+        val redisSha256StringMedArbeidsgiver = jedis.get(sha256StringMedArbeidsgiver)
+        val redisSha256StringUtenArbeidsgiver = jedis.get(sha256StringUtenArbeidsgiver)
+        if (redisSha256StringMedArbeidsgiver != null) {
             log.warn("Melding market som infotrygd duplikat oppdaatering {}", fields(loggingMeta))
+        } else if (redisSha256StringUtenArbeidsgiver != null) {
+            log.info("Arbeidsgiver er endret, send oppdatering til infotrygd")
```

Comment:
Dette funker vel hvis sykmelding uten arbeidsgiver er sendt først, men hva hvis det er sykmelding med arbeidsgiver som er sendt først..?

Reply:
Det vil funke for begge deler, vertfall ifølge testene eg har lagd